### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by amita-roy**
- I'm working on the testing branch to add tests for the math-magicians website.
- All the tests are in the src/test directory.
- Test has been added via jest and a snapshot for all three routes has also been added.
- For randomly generated quotes,  I have used jest mock for quotes route.

## In the milestone1 pull request  I have declared that I was having an issue with stylelint so I have to make changes and mentioned that what changes I'm making, after that my other milestones also got approved with that change.

### CODE REVIEW

**Review by krys2fa — CHANGES_REQUESTED** [review]
# Status: Marked as Invalid! ⚠️  
Please note that your Code Review Request is being marked as invalid. 
This is because the configs in .github/workflows/ must be exactly the same as that in the [Microverse linter config repo](https://github.com/microverseinc/linters-config/blob/master/react-redux/.github/workflows/linters.yml). 

💡 Do not make any changes in config files - they represent style guidelines that you share with your team - which is a group of all Microverse students.
If you think that the change is necessary - open a [Pull Request in this repository](https://github.com/microverseinc/linters-config/blob/master/README.md#contributing).


This Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready!

**Comment by amita-roy** [comment]
### In the milestone1 pull request, I have declared that I was having an issue with stylelint so I have to make changes and mentioned that what changes I'm making, after that my other milestones also got approved with that change.

File: src/test/calculate.test.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
+import Calculate from '../logic/calculate';
```
Comment (Torres-ssf) [inline src/test/calculate.test.js:1]: - [x] It seems the modifier operator `%` is not being tested. We need to implement at least a test for it.

File: src/test/calculatorpage.test.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  });
+  test('has button +/- in the document', () => {
+    render(<Calculator />);
+    const element = screen.getByText('+/-');
+    expect(element).toBeInTheDocument();
+  });
+  test('has element with className calculator', () => {
+    render(<Calculator />);
+    const { container } = render(<div className="calculator" />);
+    // const element = screen.getByRole('display');
```
Comment (Torres-ssf) [inline src/test/calculatorpage.test.js]: - [x] Let's remove this commented code.

**Review by Torres-ssf — CHANGES_REQUESTED** [review]
### Changes required :muscle: 
Hi @Amita-Roy :hand: 
Awesome job on testing your application so far.
But we need to make so tests before moving on.

- [x] We have one test failing, we need to fix it.

Happy coding :computer: 
**Sérgio Torres**

**Comment by amita-roy** [comment]
- Added a test for % operator
- Removed the commented out code
- All tests are passing, please don’t refer to the below screenshot

Screenshot 2021-03-28 at 11 59 52 AM

**Review by Torres-ssf — APPROVED** [review]
### PR Approved :clap: :tada: 
Hello again @Amita-Roy :hand: 
Great, all required changes were implemented.
By the way, your calculator style is awesome, it's looking really nice.
Good luck with the next project

Happy coding :computer: 
**Sérgio Torres**

**Pull request merged**